### PR TITLE
A few minor tweaks that have come out of snoo's day projects

### PR DESCRIPTION
**Pull request by spladug**
:eyeglasses: @dellis23 @pacejackson

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -42,7 +42,14 @@
 
 setup(
     name="baseplate",
+    description="A library to build services on",
+    long_description=open("README.rst").read(),
+    author="Neil Williams",
```
Comment (dellis23) [inline setup.py]: can we put my name here instead

**Comment by pacejackson** [comment]
Neat, lgtm 🐟

File: setup.py (hunk 1 above)
Comment (spladug) [inline setup.py]: i actually don't really know what to put here. should it be `reddit inc`?

Comment (spladug) [inline setup.py]: otherwise, yeah, we could put your name here, or maybe putting `Albert Einstein` would lend us some cred

**Comment by enmatt** [comment]
Looks good.

**Comment by siemiatj** [comment]
Good to merge

**Comment by spladug** [comment]
Thanks for all the support, Dan's friends!

**Pull request merged**